OSDOCS-13805: Kueue 1.0 release notes (#95725)
openshift/openshift-docs: modules/release-notes-1.0.adoc

// Module included in the following assemblies:
//
// * release_notes/release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="release-notes-1.0_{context}"]
= Release notes for {product-title} version 1.0

{product-title} version 1.0 is a generally available release that is supported on {platform} versions 4.18 and 4.19.

{product-title} version 1.0 uses link:https://kueue.sigs.k8s.io/docs/overview/[Kueue] version 0.11.

[id="release-notes-1.0-new-features"]
== New features in {product-title} version 1.0

The following features are supported in {product-title} version 1.0:

* Role-based access control (RBAC) enables you to control which types of users can create which types of {product-title} resources.

* Configuring resource quotas by creating cluster queues, resource flavors, and local queues enables you to control the amount of resources used by user-submitted jobs and workloads.

> * Labeling namespaces and configuring label policies enables you to control which jobs and workloads are managed by {product-title}.

Reviewer: Maybe to avoid translation errors:
```suggestion
* Labeling namespaces and configuring label policies enables you to control the jobs and workloads that are managed by {product-title}.
```

> * Labeling namespaces and configuring label policies enables you to control which jobs and workloads are managed by {product-title}.

Author: Not really, it's a couple of different config settings that are used for similar purposes, so it makes sense to group them

> * Creating cohorts and configuring fair sharing and gang scheduling settings enables you to share unused, borrowable resources between queues.

Reviewer: > Creating cohorts and configuring fair sharing and gang scheduling settings

This sounds like two features: creating and configuring.

```suggestion
* Creating cohorts and configuring fair sharing and gang scheduling settings enable you to share unused, borrowable resources between queues.
```

[id="release-notes-1.0-known-issues"]
== Known issues in {product-title} version 1.0

* {product-title} uses the `managedJobsNamespaceSelector` configuration field, so that administrators can configure which namespaces opt in to be managed by {product-title}. Because namespaces must be manually configured to opt in to being managed by {product-title}, resources in system or third-party namespaces are not impacted or managed by {product-title}.
+

> The current behavior in {product-title} allows reconciliation of `Job` resources that have the `kueue.x-k8s.io/queue-name` label, even if these resources are in namespaces which are not configured to opt in to being managed by {product-title}. This is inconsistent with the behavior for other core integrations like pods, deployments, and stateful sets, which are only reconciled if they are in namespaces which have been configured to opt in to being managed by {product-title}. (link:https://issues.redhat.com/browse/OCPBUGS-58205[OCPBUGS-58205])

Reviewer: Per ISG: [Use that, without a comma, to introduce a restrictive clause](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#t). Use which, preceded by a comma, to introduce a nonrestrictive clause.
```suggestion
The current behavior in {product-title} allows reconciliation of `Job` resources that have the `kueue.x-k8s.io/queue-name` label, even if these resources are in namespaces that are not configured to opt in to being managed by {product-title}. This is inconsistent with the behavior for other core integrations like pods, deployments, and stateful sets, which are only reconciled if they are in namespaces that have been configured to opt in to being managed by {product-title}. (link:https://issues.redhat.com/browse/OCPBUGS-58205[OCPBUGS-58205])
```
